.github/workflows/publish.yaml

```diff
@@ -137,12 +137,14 @@ jobs:
       - name: Open PR
         # We need to git push -u otherwise gh will prompt
         # asking where to push the branch.
+        env:
+          VERSION: ${{ env.VERSION }}
```

Comment:
If `VERSION` is already in `env` do we need to set it on `env` here?

Reply:
It's a pattern I've been following because I thought I had to but to be honest, it may be redundant 🤔 I feel like I had to do this to make sure it's there but haven't tested in a while!